### PR TITLE
Prepare support of PHP 8.2

### PR DESCRIPTION
**Pull request by JamesHemery**
Hi !

Just a small PR to prepare for PHP 8.2 support which we are already using in development on some projects.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ jobs:
         php: [ 7.2, 7.3, 7.4, '8.0', 8.1 ]
         laravel: [ 6, 7, 8, 9 ]
         stability: ['prefer-lowest', 'prefer-stable']
+        include:
+          - php: 8.2
+            laravel: 9
+            stability: prefer-stable --ignore-platform-req=php+
```
Comment (driesvints) [inline .github/workflows/tests.yml:21]: This line isn't needed anymore. PHP 8.2 is stable enough to start requiring other packages to release compatible versions. We've also removed this from the Laravel framework build and have a passing build. In fact, 8.2 can be added to the list of PHP versions a bit above here. It needs to be excluded for Laravel 6, 7 & 8 in the exclude list below.

**Review by driesvints — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -15,6 +15,10 @@ jobs:
         php: [ 7.2, 7.3, 7.4, '8.0', 8.1 ]
```
Comment (reinink) [inline .github/workflows/tests.yml:15]: @driesvints like so?

```suggestion
        php: [ 7.2, 7.3, 7.4, '8.0', 8.1, 8.2 ]
```

File: .github/workflows/tests.yml (hunk 1 above)
Comment (reinink) [reply .github/workflows/tests.yml:21]: Just this line (line 21)? Or more?

**Comment by driesvints** [comment]
Preparing a new PR because I can't suggest commits in here on parts that I want to change: https://github.com/inertiajs/inertia-laravel/pull/463

**Pull request closed without merging**